bitmap.go

```diff
@@ -203,16 +205,35 @@ func resize(capacity, v int) int {
 
 // MarshalJSON returns encoded string representation for the bitmap
 func (dst Bitmap) MarshalJSON() ([]byte, error) {
-	return json.Marshal(dst.ToBytes())
+	size := len(dst)
+	vals := make([]string, size)
```

Comment:
That doesn't seem efficient, allocate a single string instead of having an intermediate slice, and then write directly into the output.